parm/archive/gefs_atmos.yaml.j2

```
@@ -0,0 +1,43 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".t" + cycle_HH + "z." %}
+
+gefs:
+    name: "GEFS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_atmos.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set file_set = [] %}
+{% set membr = ["ensstat"] %}
+{% for mem in range(1, NMEM_ENS + 1) %}
+    {% do membr.append("mem{{ '%03d' % mem }}") %}
+{% endfor %}
+
+{% for mem in membr %}
+    {% for res in ['0p25', '0p50', '1p00'] %}
+        {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                              '${RUN}':RUN,
+                              '${YMD}':cycle_YMD,
+                              '${HH}':cycle_HH,
+                              '${GRID}':res,
+                              '${MEMDIR}':mem }) %}
+
+        {% set COMIN_ATMOS_GRIB = COM_ATMOS_GRIB_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+        # Select pgrb and grib files to copy to the atardir
+        {% if RUN == "gefs" %}
+            {% if path_exists(COMIN_ATMOS_GRIB) %}
+                {% for fhr in range(FHMIN_GFS, FHMAX_GFS + FHOUT_GFS, FHOUT_GFS) %}
+                    {% if membr=="ensstat" %}
+                        {% set file_name = head ~ "mean.pres_." ~ res ~ ".f" ~ '%03d'|format(fhr) ~ ".grib2" %}
+                    {% else%}
+                        {% set file_name = head ~ "pgrb2." ~ res ~ ".f" ~ '%03d'|format(fhr) %}
```

After looking more closely at the variables in pgrb2 and pgrb2b, it may be best to archive both pgrb2 and pgrb2b because variables are needed from both for the reforecast. 
```suggestion
                        {% set file_name = head ~ "pgrb2" ~ res ~ ".f" ~ '%03d'|format(fhr) %}
```

---

@EricSinsky-NOAA You can check the tarball at /work/noaa/global/antonf/archive_rotdir/gefs1/2021032312 in HERCULES. If you need to change the files or add more let me know.

---

Thanks, @AntonMFernando-NOAA. I just tested the archive task and the atmos data on HPSS looks ok. I am checking the other components.  I am also testing the archive task with a GEFS replay case.